redis-utils-lib/dev/user.clj

```diff
@@ -0,0 +1,17 @@
+(ns user
```

Comment:
Why do we need a `user` namespace here?

Reply:
The repl loads the user namespace so we can load some convenience namespace's that we only want for dev.